reason.json

```diff
@@ -15,7 +15,8 @@
     "@opam/merlin-extend": " >= 0.6",
     "@opam/result": "*",
     "@opam/ppx_derivers": "< 2.0.0",
-    "@opam/dune": "2.9.3"
+    "@opam/dune": "2.9.3",
```

Comment:
let's try to use the same dune version across all `.json` files.

Reply:
Updated in 22e86a686dbd893821f20fea8f04d3d69994e464 + update reason.json/rtop.json